kubetest/e2e.go

```diff
@@ -762,18 +762,24 @@ func kubemarkGinkgoTest(testArgs []string, dump string) error {
 }
 
 // Brings down the kubemark cluster.
-func kubemarkDown(err *error, wg *sync.WaitGroup, provider, dump string) {
+func kubemarkDown(err *error, o.logexporterGCSPath, wg *sync.WaitGroup, provider, dump string) {
 	defer wg.Done()
 	control.XMLWrap(&suite, "Kubemark MasterLogDump", func() error {
 		logDumpPath := logDumpPath(provider)
-		cmd := exec.Command(logDumpPath, dump)
 		masterName := os.Getenv("MASTER_NAME")
+		var cmd *exec.Cmd
 		cmd.Env = append(
 			os.Environ(),
 			"KUBEMARK_MASTER_NAME="+masterName,
 			"DUMP_ONLY_MASTER_LOGS=true",
 		)
-		log.Printf("Dumping logs for kubemark master: %s", masterName)
+		if logexporterGCSPath != ""{
+			log.Printf("Dumping logs from kubemark master to GCS directly at path: %v", logexporterGCSPath)
+			cmd = exec.Command(logDumpPath, dump, logexporterGCSPath)
+		}else{
+			log.Printf("Dumping logs locally to: %v", dump)
```

Comment:
Can we include "kubemark master" in the log?

Like 
log.Printf("Dumping logs for kubemark master locally: %s", masterName)

Reply:
I we will use log.Printf("Dumping logs for kubemark master locally: %s", masterName), the path to which the logs are dumped will not be printed. Can you please confirm me on that we need info of master name instead of the path in both the cases. By the way, mistakes you pointed out in the latter review are fixed.

Reply:
Do you want log.Printf("Dumping logs for kubemark master locally to: %v", dump) instead.

Reply:
Do you want log.Printf("Dumping logs for kubemark master locally to: %v", dump) instead.